anda/devs/micro/micro-nightly.spec

```diff
@@ -0,0 +1,114 @@
+# Generated by go2rpm 1.15.0
+%bcond check 0
+%bcond bootstrap 0
+
+%if %{with bootstrap}
+%global debug_package %{nil}
+%endif
+
+%if %{with bootstrap}
+%global __requires_exclude %{?__requires_exclude:%{__requires_exclude}|}^golang\\(.*\\)$
+%endif
+
+%global commit 5a62a8ead4af741e4fc5f850fcb846974859c5a0
+%global commit_date 20250209
+%global shortcommit %{sub %{commit} 1 7}
+
+# https://github.com/zyedidia/micro
+%global goipath         github.com/zyedidia/micro
+Version:                2.0.14^%{commit_date}git.%{shortcommit}
```

Comment:
maybe use a global macro for `2.0.14` and add to the if block inside update.rhai?